src/components/AccountSettings.tsx

```diff
@@ -1,330 +1,277 @@
-        } else if(verificationStateEnum === 'REJECTED' || verificationStateEnum ==='SUSPENDED'){
-            return (
-                <div className='ValidationStateContainer'>
-                    <Typography variant='headline3'>
-                        {verificationStateEnum === 'REJECTED' ? REJECTED_TEXT : SUSPENDED_TEXT}
-                    </Typography>
-                    <Typography variant='body1' >
-                        {verificationState?.reason}
-                    </Typography>
-                    <VerifyButton/>
-                </div>
-            )
-        } else {
-            return <VerifyButton/>
-        }
-    }
+  useEffect(() => {
+    getUserData()
+  }, []
+  )
 
-    // Closes any forms and resets the fields.
-    const cancelEdit = () => {
-        setEditUsername(false)
-        setUpdatedUsername('')
-        setChangePW(false)
-    }
+  const handleScroll = (ref:React.RefObject<HTMLDivElement>) => {
+    ref.current?.scrollIntoView({behavior:'smooth'})
+  }
 
-    interface EditFieldProps {
-        label: string, 
-        updatedValue: string, 
-        updateFn: Function,
-    }
-    
-    // Component to edit fields (username / email)
-    const EditField: React.FC<EditFieldProps> = ({label, updatedValue, updateFn}) => {
-        return(
-            <div>
-                <FormGroup className='required'>
-                    <FormLabel>{label}</FormLabel>
-                    <FormControl
-                        onChange={e => updateFn(e.target.value)}
-                        style={{maxWidth:'320px'}}
-                        value = {updatedValue}/>
-                    <Button className='btn-container emptyButton' onClick={cancelEdit}>
-                        Cancel
-                    </Button>
-                    <Button className='btn-container'  variant='secondary' onClick={onUpdateUserProfile}>
-                        Save Changes
-                    </Button>
-                </FormGroup>
+  return (
+    <div className='panel-wrapper-bg with-account-setting'>
+      <nav className='account-setting-panel nav-panel'>
+        <Button onClick={() => handleScroll(profileInformationRef)} variant='outlined'>Profile Information</Button> 
+        <Button onClick={() => handleScroll(changePasswordRef)} variant='outlined'>Change Password</Button>
+        <Button onClick={() => handleScroll(trustCredentialRef)} variant='outlined'>Trust & Credentials</Button>
+        <Button onClick={() => handleScroll(personalAccessTokenRef)} variant='outlined'>Personal Access Tokens</Button>
+      </nav>
+
+      <div>
+        <div ref={profileInformationRef} className='account-setting-panel main-panel'>
+          <h3>Profile Information</h3>
+          <p>This information is reused across all Sage products. Lorem ipsum dolor sit amet consectetuer adapiscing elit, sed do euismod tempore incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam. Quis nostrud exercitation ullamco laboris.</p>
```

Comment:
This copy is from the design which needs to be updated.

Reply:
Could you just remove the "lorem Ipsum..." for now?